### PR TITLE
message is displayed if security is not active

### LINKED ISSUE #2383: security activity issue
**Actual Behaviour**

security activity functionalities becomes irresponsive if password is not set. 

**Expected Behaviour**

a toast should be displayed if no password is set when the user taps on any of the functionalities except access by password.

**Steps to reproduce it**

go to settings, navigate to security. disable access by password. tap on any other functions like change by password. action has no response.

**Screenshots of the issue**

whatsapp image 2019-01-12 at 23 39 24

**Would you like to work on the issue?**

yes.

### PR DESCRIPTION
**Pull request by kk63852**
Fixed #2383 

Changes: 
hard-coded strings were added to string xml.
a message is now displayed if security is not active.
code reformatted.

:

### CODE REVIEW

File: app/src/main/java/org/fossasia/phimpme/gallery/activities/SecurityActivity.java [hunk 1]
```diff
@@ -482,56 +510,56 @@ public void onClick(DialogInterface dialog, int which) {
...
+                                editTextConfirmPassword.setError(getString(R.string.password_dont_match));
+                            }
+                        } else {
+                            editTextPassword.requestFocus();
+                            editTextPassword.setError(getString(R.string.error_password_length));
+                        }
+                    }
+                });
+            }
+
```
Comment (GOVINDDIXIT) [inline app/src/main/java/org/fossasia/phimpme/gallery/activities/SecurityActivity.java]: extra

File: app/src/main/java/org/fossasia/phimpme/gallery/activities/SecurityActivity.java [hunk 2]
```diff
@@ -482,56 +510,56 @@ public void onClick(DialogInterface dialog, int which) {
...
+                                    if (securityAnswer1.getText().length() != 0) {
+                                        SP.putString(getString(R.string.preference_password_value), editTextPassword.getText().toString());
+                                        SP.putString(getString(R.string.security_question), securityQuestion.getText().toString());
+                                        SP.putString(getString(R.string.security_answer), securityAnswer1.getText().toString());
+                                        securityObj.updateSecuritySetting();
+                                        SnackBarHandler.show(llroot, R.string.remember_password_message);
+                                        dialog.dismiss();
+                                        Snackbar.make(findViewById(android.R.id.content), "Password Changed", Snackbar.LENGTH_SHORT)
+                                                .show();
+
```
Comment (GOVINDDIXIT) [inline app/src/main/java/org/fossasia/phimpme/gallery/activities/SecurityActivity.java]: extra

**Comment by kk63852** [comment]
i reformatted the after resolving the issue. the indentations were done by studio. @GOVINDDIXIT

**Comment by sauravvishal8797** [comment]
@kk63852 Please resolve the conflicts.

**Comment by mariobehling** [comment]
As we decided to take out this feature this PR is no longer required. Thanks for your contributions!

**Pull request closed without merging**